pcdsdevices/valve.py

```diff
@@ -128,6 +128,14 @@ def _set_lightpath_states(self, lightpath_values):
         self._removed = lightpath_values[self.open_limit]['value']
 
 
+class VRCClsLS(VVC):
```

Comment:
Looks like a lot of classes in this file aren't covered by the test suite :/

Reply:
We should have some overall tests that try to make every device
There's not much meaningful testing that we can do if the classes are just PV definitions

Reply:
We kinda do: https://github.com/pcdshub/pcdsdevices/blob/39a9dca3adaf8b46a2f5958084aee87948e0f99f/pcdsdevices/tests/test_interface.py#L271-L281